lib/v2/mangadex/index.js

```diff
@@ -0,0 +1,82 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const { RequestInProgressError } = require('@/errors');
+
+module.exports = async (ctx) => {
```

Comment:
It will look nicer if you declare `id` and `lang` vs using `ctx.params.id` and so every time
```suggestion
module.exports = async (ctx) => {
    const { id, lang } = ctx.params;
```